app/views/businesses/show.html.erb

```diff
@@ -3,7 +3,7 @@
   <%= @business.description %>
 </div>
 
-<% if current_admin? || platform_admin? %>
+<% if (current_admin? && @business.admins.include?(current_user)) || platform_admin? %>
     <%= link_to "View Business Dashboard", business_dashboard_path(@business.slug) %>
```

Comment:
hmm, since this is used more than once we could probably pull this out into its own method
if (current_admin? && @business.admins.include?(current_user))